### ISSUE TITLE
ciritical bug: cmap folder should be included in assets folder! 

### ISSUE THREAD

**Issue by leo6104** [stephanrauh/ngx-extended-pdf-viewer#208]
**Check this first**
- Does the bug also happen on https://mozilla.github.io/pdf.js/web/viewer.htm? No.
- Does the bug also happen on https://pdfviewer.net/simple? Yes

**Describe the bug**

Some pdf include special character that doesn't support by pdf.js as default. 
To support the character, pdf.js load cmap file lazily. 

You can set cmap folder by `defaultOptions.cMapUrl`.

But in ngx-extended-pdf-viewer, it does not include cmap folder and doesn't include documentation how to set cmap folder.

I suggest that we should include https://github.com/mozilla/pdfjs-dist/tree/master/cmaps files in node_modules/ngx-extended-pdf-viewer/assets/cmaps/ folder.
and we should add the documentation to make developers 

add to angular.json
```
              {
                "glob": "**/*",
                "input": "node_modules/ngx-extended-pdf-viewer/assets/cmaps",
                "output": "/assets/cmaps/"
              },
``` 

add `defaultOptions.cMapUrl = './assets/cmaps/';` in their code. 

**Version info**
- Version of ngx-extended-pdf-viewer: 1.8.4 or 2.0.0-alpha.x

**Demo PDF file**
Sorry for not providing PDF. its problem was occured with my client's website so the pdf was commercial usage. I cannot share you the pdf. But it is problem that we not include cmap files.

**Additional context**
- https://github.com/mozilla/pdf.js/issues/7927
- https://github.com/mozilla/pdfjs-dist/tree/master/cmaps
- https://stackoverflow.com/questions/32764773/what-is-a-pdf-bcmap-file

**Comment by leo6104** [comment]
In my project, i fix the cmap files issue with these step.

```
npm install pdfjs-dist -D
```

Add to angular.json "scripts" part
```
               {
                "glob": "**/*",
                "input": "node_modules/pdfjs-dist/cmaps",
                "output": "/assets/cmaps/"
              },
```

Add to typescript (where i add ngx-extended-pdf-viewer)
```
    defaultOptions.cMapUrl = './assets/cmaps/';
```

It solve the issue with my project.

**Comment by stephanrauh** [comment]
No problem. I've just published version 2.0.0-alpha.8 containing the CMAP files and the default configuration to find them automatically. Would you mind to test the new version?

You know, I've been investigating the topic of CMAP files several times. The only reason I didn't add them earlier was I didn't know how to test them. I still don't. I've learned CMAP files are primarily needed by old PDF documents written in Korean, Chinese or Japanese. However, I haven't been able to find such a file. Every file I've found is displayed correctly without the CMAP files.

So I'd appreciate your help. Hope the new version fixes your problems. You can drop the `defaultOptions` bit, I've made the correct folder the default.

Best regards,
Stephan

**Comment by leo6104** [comment]
Thank you for your quick response & publish. It works! :) 

![image](https://user-images.githubusercontent.com/7777929/74398026-95c51b00-4e59-11ea-97af-7156fd5bde6b.png)
